packages/components/forms/src/BaseCheckbox/BaseCheckbox.styles.ts

```diff
@@ -48,6 +48,9 @@ const getStyles = ({
       position: 'absolute',
       width: tokens.spacingM,
       zIndex: tokens.zIndexDefault,
+      '+ span': {
+        minWidth: tokens.spacingM,
```

Comment:
I still dislike reading a size property with a spacing token 😅 
But I know that's how we use them

Reply:
I agree, and it is implemented for a vast amount of components in the design system.